libethcore/Transaction.cpp

```diff
@@ -31,6 +31,12 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
+namespace
+{
+int const c_chainIdInNonceBits = 64;
+u256 const c_nonceLowMask = 0xffffffffffffffff;
```

Comment:
So I checked :) https://gist.github.com/chfast/dc7cfb070947f703e57c25dc3cbb79f7.

- Both versions are very similar, they call `eval_bitwise_...` template function (big unrolled code in the middle).
- `u256 c_nonceLowMask` does not require dynamic initialization. See `_ZN12_GLOBAL__N_1L10lowMask256E`.
- The version with `uint64_t` is a bit better because does not require global constant loading.

That was not to prove you ware wrong, I was really curious. I would slightly prefer native type in the constant, but you can decide. Let me know.